public/data/glTF-projects-data.json

```diff
@@ -2160,5 +2146,51 @@
   "type" : [ "application" ],
   "language" : [ "c++" ],
   "inputs" : [ "glTF 2.0" ]
+}, {
+  "name": "Cesium for Unreal",
+  "link": "https://cesium.com/platform/cesium-for-unreal/",
+  "description": "A plugin for the Unreal Engine that allows streaming assets from the cloud, using the 3D Tiles format that is based on glTF",
+  "task": ["view"],
+  "type": ["plugin"],
+  "license": ["Apache-2.0"],
+  "language": ["C++"],
+  "inputs": ["3D Tiles", "glTF 2.0"]
+}, {
+  "name": "Cesium Native",
+  "link": "https://github.com/CesiumGS/cesium-native",
+  "description": "A base library for 3D geospatial applications, including lightweight glTF serialization and deserialization",
+  "task": ["load", "import", "export"],
+  "type": ["library"],
+  "license": ["Apache-2.0"],
+  "language": ["C++"],
+  "inputs": ["glTF 2.0"],
+  "outputs": ["glTF 2.0"]
+}, {
+  "name": "CesiumJS",
+  "link": "https://cesium.com/platform/cesiumjs/",
+  "description": "A library for creating geospatial web applications, and visualize data in the 3D Tiles format that is based on glTF",
+  "task": ["view"],
+  "type": ["library", "application"],
+  "license": ["Apache-2.0"],
+  "language": ["JavaScript", "TypeScript"],
+  "inputs": ["glTF 1.0", "glTF 2.0"]
+}, {
+  "name": "Cesium ion",
+  "link": "https://cesium.com/platform/cesium-ion/",
+  "description": "A platform for hosting, optimizing, and streaming massive amounts of geospatial data, using the 3D Tiles format that is based on glTF",
+  "task": ["import", "optimize"],
+  "type": ["web-api"],
+  "inputs": ["glTF 1.0", "glTF 2.0", "FBX", "CityGML", "CZML", "KML", "LAS", "LAZ", "DAE", "OBJ", "FLT", "ASC", "SRC", "TIFF", "DEM", "JPG", "PNG", "TERRAINDB"],
+  "outputs": ["3D Tiles", "glTF 2.0"]
+}, {
+  "name": "Cesium gltf-pipeline",
```

Comment:
Cesium started this project (in collaboration with one of my former students), but I would not call it "Cesium gltf-pipeline"; it is just "glTF Pipeline"